pkg/controller/etcdproxy/helpers.go

```diff
@@ -53,9 +53,10 @@ func newReplicaSet(etcdstorage *etcdstoragev1alpha1.EtcdStorage,
 								"--cacert=/etc/coreetcd-certs/ca/ca.pem",
 								"--cert=/etc/coreetcd-certs/client/client.pem",
 								"--key=/etc/coreetcd-certs/client/client-key.pem",
-								"--trusted-ca-file=/etc/etcdproxy-certs/ca/ca.pem",
-								"--cert-file=/etc/etcdproxy-certs/server/server.pem",
-								"--key-file=/etc/etcdproxy-certs/server/server-key.pem",
+								"--trusted-ca-file=/etc/etcdproxy-certs/ca/client-ca.crt",
+								"--cert-file=/etc/etcdproxy-certs/server/tls.crt",
+								"--key-file=/etc/etcdproxy-certs/server/tls.key",
+								"--debug=true",
```

Comment:
intentional?

Reply:
No, I needed it while debugging etcd problems, but don't need it anymore. Removed.